ci/Jenkinsfiles/build.groovy

```diff
@@ -230,7 +230,7 @@ pipeline {
     TEST_SERVICE_DOMAIN_SUFFIX = 'svc.cluster.local'
     SLACK_CHANNEL = 'platform-notifs'
     CONNECT_PREPROD_URL = 'https://nos-preprod-connect.nuxeocloud.com/nuxeo'
-    NUXEO_PACKAGE_PATH = 'nuxeo-compound-documents-package/target/nuxeo-compound-documents-package-*.zip'
+    NUXEO_PACKAGE_PATH = 'nuxeo-compound-documents-package/target/nuxeo-compound-documents-package-$VERSION.zip'
```

Comment:
```suggestion
    NUXEO_PACKAGE_PATH = "nuxeo-compound-documents-package/target/nuxeo-compound-documents-package-$VERSION.zip'"
```
to have Groovy interpolate the version directly, rather than relying on shell interpolation at call time (and not sure about interpolation in `archiveArtifacts allowEmptyArchive: true, artifacts: "${NUXEO_PACKAGE_PATH}"`)

So, `VERSION = getVersion()` need to be defined first.